clients/client-accessanalyzer/commands/ApplyArchiveRuleCommand.ts

```diff
@@ -23,6 +23,18 @@ export interface ApplyArchiveRuleCommandOutput extends __MetadataBearer {}
 /**
  * <p>Retroactively applies the archive rule to existing findings that meet the archive rule
  *          criteria.</p>
+ * @example
+ * User a bare-bone client and the command you need to make an API call.
```

Comment:
Diff is too large, so can't comment on codegen file. It should be `bare-bones`
```suggestion
 * User a bare-bones client and the command you need to make an API call.
```

Reply:
Updated: 
![Screen Shot 2021-05-03 at 3 50 13 PM](https://user-images.githubusercontent.com/7614947/116942908-5aae3380-ac27-11eb-88c8-8ef0f1a60f73.png)